### ISSUE TITLE
Suggestion for MobileNetV2

### ISSUE THREAD

**Issue by nlp-tokio** [kuangliu/pytorch-cifar#133]
I found that  the MobileNetv2 has more shortcut with conv1d and batchnorm ops when stride==1 compared to original paper, which leads to the high performance of the modified model. I think extra comment could be added into MobileNetv2 code file for other guys.
